FirebaseRemoteConfig/Tests/Unit/RCNConfigDBManagerTest.m

```diff
@@ -193,7 +193,8 @@ - (void)testWriteAndLoadMetadataResult {
   NSString *bundleIdentifier = [NSBundle mainBundle].bundleIdentifier;
   NSTimeInterval lastFetchTimestamp = [NSDate date].timeIntervalSince1970;
 
-  NSDictionary *deviceContext = @{@"app_version" : @"1.0.1", @"os_version" : @"iOS9.1"};
+  NSDictionary *deviceContext =
```

Comment:
Please add tests for `FIRRemoteConfigAppVersion` and `FIRRemoteConfigShortVersion`

Reply:
Discussed offline.